app/src/main/java/com/droidhats/campuscompass/MerlinActivity.kt

```diff
@@ -0,0 +1,76 @@
+package com.droidhats.campuscompass
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.novoda.merlin.*
+
+/**
+ * This class has the objective of outlining the required methods for using the Merlin library
+ * The library is used to observe the network connection status while using the application
+ */
+abstract class MerlinActivity : AppCompatActivity() {
+
+    protected var merlin: Merlin? = null
+
+    /**
+     * Overrides the activity's OnCreate method to attach the Merlin object
+     *
+     * @param Bundle: the saved state of the application to pass between default Android methods
```

Comment:
I think you meant `savedInstanceState `instead of `Bundle` 😉

Reply:
Fixed